db/bigtable_eth1.go

```diff
@@ -3057,8 +3079,14 @@ func (bigtable *Bigtable) GetAddressNames(addresses map[string]string) error {
 
 	keys := make([]string, 0, len(addresses))
 
-	for address := range addresses {
-		keys = append(keys, fmt.Sprintf("%s:%x", bigtable.chainId, address))
+	if err := GetEnsNamesForAddress(addresses); err != nil {
+		return err
+	}
```

Comment:
The context created here
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/c3c3511cf877ac4146c9fb1080251416f01278c7/db/bigtable_eth1.go#L3077-L3078
is not needed for this ENS block and should therefore be created below the ENS block.

Reply:
We usually create the context at the start so I will leave it there.